ui/revenuecatui/src/testDebug/kotlin/com/revenuecat/purchases/ui/revenuecatui/snapshottests/PaywallComponentsTemplatePreviewRecorder.kt

```diff
@@ -39,7 +39,7 @@ class PaywallComponentsTemplatePreviewRecorder internal constructor(
     companion object {
         @JvmStatic
         // Placing the offering ID in inverted double diamond brackets so we can easily parse it later.
-        @Parameters(name = ">>{0}<<")
+        @Parameters(name = "{0}")
```

Comment:
Paparazzi started complaining about this saying these characters were unsupported in the latest version... We will need to store a new set of snapshots with this change unfortunately.